core/src/main/groovy/noe/server/ServerAbstract.groovy

```diff
@@ -626,6 +627,8 @@ abstract class ServerAbstract implements IApp {
             }
           }
 
+          log.debug('Filtering expected warning due to lack of entropy at virtualized environment.')
```

Comment:
Why we need to log that ?

Reply:
At least some information should be present in debug log about the filtering in case of any investigation.